src/main/protobuf/soc/gameState.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
+
+package protos.soc;
+
+import "soc/board.proto";
+import "soc/inventory.proto";
+import "soc/playerState.proto";
+
+message GameState {
+    required Board board = 1;
+    repeated PlayerState player_states = 2;
+    required int32 num_development_cards_left = 3;
+    required Player current_player = 4;
+    required bool can_roll_dice = 5;
```

Comment:
I didn't explain this well in the previous message, but I really think these two fields should be removed in favor of a `TURN_PHASE` enum, which currently has `ROLL_PHASE` and `MAIN_PHASE`. (I envision a potential future in which we have a `TRADE_PHASE`) 

As to the issue of the client needing to know it can play a development card. The client knows when it bought its development cards, and also what turns it played them. It can resolve the logic of if it can play a development card itself:
```
has_played_card_this_turn = any([card.turn_played == current_turn_number for card in played_dev_cards])
has_card_to_play = any([card.turn_bought < current_turn_number for card in unplayed_dev_cards])
can_play_development_card = not has_layed_card_this_turn and has_card_to_play
```

Reply:
I do not think the information of when the client bought the development card is relevant and should be included. It is not public information would only be relevant to the point of suggesting that a player can play that card on its turn. (Also is the turn number even info we are sending to the client? )